src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js

```diff
@@ -169,10 +169,11 @@
     $(".page").hide();
     $("#stackpage").show();
 
-    $("#rawExceptionButton").click(function () {
+    $("#rawExceptionButton").click(function (event) {
         var div = document.getElementById('rawException');
         div.style.display = 'inline-block';
         div.scrollIntoView(true);
+        event.preventDefault();
```

Comment:
Any reason to not also return false? It's been a while since I've JavaScripted :trollface:  but that's the recommended old pattern I'm used to (to do both).